charts/promitor-agent-scraper/values.yaml

```diff
@@ -76,3 +76,22 @@ resources: {}
   # requests:
   #  cpu: 100m
   #  memory: 128Mi
+
+rbac:
+  ## If true, create & use RBAC resources
+  ##
+  create: true
+
+  ## If true, create & use Pod Security Policy resources
+  ## https://kubernetes.io/docs/concepts/policy/pod-security-policy/
+  pspEnabled: false
```

Comment:
I tend to avoid having abbreviations as they are not really clear, what if we change this to `podSecurityPolicyEnabled`?